cupyx/fallback_mode/fallback.py

```diff
@@ -141,16 +169,64 @@ def __new__(cls, *args, **kwargs):
             return object.__new__(cls)
 
         cupy_ndarray_init = cp.ndarray(*args, **kwargs)
-        return cls(_stored=cupy_ndarray_init)
+        return cls(_stored=cupy_ndarray_init, _class=cp.ndarray)
 
     def __init__(self, *args, **kwargs):
+        """
+        Args:
+            _stored (None, ndarray): If _stored is None, object is not
```

Comment:
Please write `None, numpy.ndarray or cupy.ndarray` explicitly because `_stored` must not be `fallback.ndarray`.

Reply:
Right. Done